src/Routes/ImageManager/Images.js

```diff
@@ -81,6 +100,106 @@ const Images = () => {
     }),
     shallowEqual
   );
+
+  useEffect(() => {
+    if (data) {
+      setRows(
+        flatten(
+          data.map(item => {
+            const statusIconHash = {
```

Comment:
Move this to `src/Routes/ImageManager/constants.js`

Reply:
Use https://github.com/RedHatInsights/edge-frontend/blob/master/src/Routes/ImageManagerDetail/constants.js instead of statusIconHash.

Reply:
https://github.com/RedHatInsights/edge-frontend/blob/master/src/Routes/ImageManagerDetail/constants.js#L9-L23 - probably you will to update some of it but most of it is good.

Reply:
Good point on using the statusIcon mapper. We could also add classes `edge-m-unknown`, `edge-m-created`, `edge-m-building`, `edge-m-error` and `edge-m-succes` and add these classes to `App.scss` with corresponding `PF` colors.